util/getArchitecture.ts

```diff
@@ -0,0 +1,19 @@
+/// <reference types="user-agent-data-types" />
```

Comment:
don't forget to test the unit

Reply:
> We can add unit tests in a follow-up PR

I'm not very keen on this kind of practice. I think it's wiser to add the test of something when it's introduced into the codebase.

Reply:
In general, I'm also against, having that said, this is the sort of change that we need before we release the website :) so I don't want to block unnecessarily this PR.